server/chat.ts

```diff
@@ -1720,7 +1720,7 @@ export const Chat = new class {
 			if (typeof entry !== 'function') continue;
 
 			const handlerCode = entry.toString();
-			entry.requiresRoom = /\bthis\.requires?Room\(/.test(handlerCode);
+			entry.requiresRoom = /requireRoom\((?:'|"|`)(.*?)(?:'|"|`).\)/.exec(handlerCode)?.[1] || /this\.requireRoom\(/.test(handlerCode);
```

Comment:
What's with the `.\)`? There should be a cast to `RoomID` after the string, right? There's no reason to search for the closing paren anyway, right?